.github/workflows/cicd.yaml

```
@@ -57,7 +57,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Install packages
-        run: pip3 install -r requirements_dev.txt
+        run: YARL_NO_EXTENSIONS=1 MULTIDICT_NO_EXTENSIONS=1 pip3 install -r requirements_dev.txt
```

```suggestion
        # Since we run inside an alpine based container, we cannot compile yarl and multidict
        # See https://pypi.org/project/yarl/ and https://github.com/aio-libs/multidict
        run: YARL_NO_EXTENSIONS=1 MULTIDICT_NO_EXTENSIONS=1 pip3 install -r requirements_dev.txt
```
would add this as a comment to explain; they both mention performance losses of 20-50x (not % !), so maybe we'd want to benchmark that and if grave, switch to non-alpine?

---

Probably same for Dockerfile

---

@annekebr tested the performance with a `python:slim` image and there is no difference, so i think we can keep the alpine